data/transit/route/train.json

```diff
@@ -2381,11 +2370,19 @@
     },
     {
       "displayName": "Zürcher Verkehrsverbund",
-      "id": "zurcherverkehrsverbund-91ea8f",
-      "locationSet": {"include": ["ch", "de"]},
+      "id": "zurcherverkehrsverbund-44bbb3",
+      "locationSet": {"include": ["001"]},
       "tags": {
         "network": "Zürcher Verkehrsverbund",
-        "network:short": "ZVV",
+        "route": "train"
+      }
+    },
+    {
+      "displayName": "ZVV",
```

Comment:
I looked at de.wikipedia.org/wiki/Zürcher Verkehrsverbund and found the "Zürcher Verkehrsverbund" related wikidata is Q201977. And currently more than 1000+ OSM elements have a "network:wikipedia=de:Zürcher Verkehrsverbund", looks like it still in use.

Is there some different with them?

Reply:
The common name is ZVV, we always used it this way, until iD users started changing the tags based the suggestion. I'm already cleaning up the current mess, it will take a bit.